packages/flutter/lib/src/material/material.dart

```diff
@@ -76,6 +85,15 @@ abstract class MaterialInkController {
 
 /// A piece of material.
 ///
+/// The Material widget is responsible for:
+/// 1. Clipping - Material clips its widget sub-tree to the shape specified by
```

Comment:
i think markdown is going to want you to have blank lines between the paragraph and the first list item.

Reply:
done